### DISCUSSION TITLE
Rigging/Heirarchy not working

### DISCUSSION THREAD

**Discussion by Pitchblack70** [keiwando/toonsquid#401] [Q&A]
So - I've got a rig set up - and they've worked in the past.

But with this project, when I try to move something, the connection isn't there. Is that a glitch, or something I'm doing wrong? (I've looked at the parent chlid relationship, and it's correct. I also tried to re-do it, but it stayed unconnected.

FWIW, the children are symbols, and the parent an image. Could that be the problem?  And if so, how to fix or work around? 

Thanks!! :)))

**Comment by keiwando** [comment]
Make sure that you don't have the "Edit pixels" option enabled in the transform tool when you move the parent layer (if by "image" you mean pixel layer - otherwise the image is also just a symbol). If that isn't it and you have a project file where this can be reproduced, please send a copy of it to contact@toonsquid.com (or let me know so I can set up a Google Drive folder for you to upload it into if it's too large to send via mail)

**Comment by Pitchblack70** [comment]
Thanks on this tip!!  In my case, it was a vector image for the parent, and the option was "Edit Path".
But when I turned it off, everything worked, and the child symbols followed!  :)))))
